src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -227,9 +227,8 @@ bool LottieLoader::open(const char* data, uint32_t size, bool copy, const std::s
     this->copy = copy;
 
     if (!resourcePath.empty()) {
-        auto path = resourcePath.c_str();
-        auto len = strlen(path);
-        this->dirName = strDuplicate(path, len);
+        if (resourcePath.back() == '/') this->dirName = strdup(resourcePath.c_str());
```

Comment:
this patch shouldn't have this change.

Reply:
Ok, This is the last part.

Since the other platforms might not same with '/' but '\\'.  it's getting become complex. 
So, rather then appending "/" with a more complex condition here,  I'd like to suggest add it all time in tvgLottieParser.cpp (872)
```
 snprintf(image->path, len, "%s%s%s", dirName, subPath, data); ->
        snprintf(image->path, len, "%s/%s%s", dirName, subPath, data);
```